### DISCUSSION TITLE
un mode 18+

### DISCUSSION THREAD

**Discussion by paulomafio50** [skarab42/marv#121] [Ideas]
Sa serait marrant d' avoir un effet a l’ancienne canal+ crypté
![image](https://user-images.githubusercontent.com/46054544/106248171-e7a0b580-6210-11eb-9702-4ea1a8932fe5.png)

**Comment by paulomafio50** [comment]
et pkoi pas un petitprogramme  de decryptage pour les sub

**Comment by mirobane** [comment]
quitte a être sérieux dans le trip, plus pour un plugin OBS couplé a un user plugin pour navigateur non ?
sinon, vous pouvez avoir un simulacre de cet effet avec ces plugins OBS comme expliqué ici https://www.youtube.com/watch?v=NuWKrpNCE1k :
 
► Dynamic Delay https://bit.ly/2ZvTyWG
► OBS ShaderFilter https://bit.ly/3s4ngym
► VHS Shader https://bit.ly/3bsOI1U
